music/src/core/visualizer.ts

```diff
@@ -438,8 +439,13 @@ export abstract class BaseSVGVisualizer extends BaseVisualizer {
         ['isDrum', note.isDrum === true],
         ['pitch', note.pitch],
       ];
+      const cssProperties: CSSProperty[] = [
+        ['--midi-velocity',
```

Comment:
Just to triple check I understand how this works: this property will be set on each `rect.note`, and will be different for each rect, right? the velocity is definitely different for each note, so I want to make sure that is reflected

Reply:
Yes, exactly. Every `rect` will basically get a `style="--midi-velocity: ..."`.